src/content/en/updates/posts/2016/02/devtools-digest-supercharged-remote-debugging.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: updates/post
+title: "Supercharged Remote Rebugging, Class Toggles and our own late night show?!"
+published_on: 2016-02-25
+updated_on: 2016-02-25
+authors:
+  - pbakaus
+tags:
```

Comment:
I was asking if these tags also work on web/tools, but I just checked the site and it looks like the tags are only displayed in the web/updates section.